scripts/pyparser/data/ecs_computer-science.json

```diff
@@ -0,0 +1 @@
tism and Waves", "course": "PHYS 2326", "LD": 3, "UD": "N/A"}, {"title": "Physics Laboratory II", "course": "PHYS 2126", "LD": 1, "UD": "N/A"}], "MAJOR CORE REQUIREMENTS": [{"title": "Professional Responsibility in CS/SE", "course": "CS 3162", "LD": "N/A", "UD": 1}, {"title": "Discrete Math for Computing II", "course": "CS 3305", "LD": "N/A", "UD": 3}, {"title": "Probability & Statistics in CS/SE", "course": "CS 3341", "LD": "N/A", "UD": 3}, {"title": "Data Structures & Intro to Algorithmic Analysis", "course": "CS 3345", "LD": "N/A", "UD": 3}, {"title": "Software Engineering", "course": "CS 3354", "LD": "N/A", "UD": 3}, {"title": "C/C++ Programming in a UNIX Envirn", "course": "CS 3377", "LD": "N/A", "UD": 3}, {"title": "Organization of Programming Lang", "course": "CS 4337", "LD": "N/A", "UD": 3}, {"title": "Digital Logic and Computer Design", "course": "CS 4341", "LD": "N/A", "UD": 3}, {"title": "Digital Systems Laboratory", "course": "CS 4141", "LD": "N/A", "UD": 1}, {"title": "Database Systems", "course": "CS 4347", "LD": "N/A", "UD": 3}, {"title": "Operating Systems Concepts", "course": "CS 4348", "LD": "N/A", "UD": 3}, {"title": "Adv Algorithm Design & Analysis", "course": "CS 4349", "LD": "N/A", "UD": 3}, {"title": "Automata Theory", "course": "CS 4384", "LD": "N/A", "UD": 3}, {"title": "Computer Science Project", "course": "CS 4485", "LD": "N/A", "UD": 4}, {"title": "CS Guided Elective", "course": "CS 43XX", "LD": "N/A", "UD": 3}, {"title": "CS Guided Elective", "course": "CS 43XX", "LD": "N/A", "UD": 3}, {"title": "CS Guided Elective", "course": "CS 43XX", "LD": "N/A", "UD": 3}, {"title": "Professional & Technical Communication", "course": "ECS 3390", "LD": "N/A", "UD": 3}], "REMAINING CORE CURRICULUM": [{"title": "Rhetoric", "course": "RHET 1302", "LD": 3, "UD": "N/A"}, {"title": "American National Government", "course": "GOVT 2305", "LD": 3, "UD": "N/A"}, {"title": "State and Local Government", "course": "GOVT 2306", "LD": 3, "UD": "N/A"}, {"title": "US History Survey to Civil War (Core: 060)", "course": "HIST 1301", "LD": 3, "UD": "N/A"}, {"title": "US History Survey from Civil War (Core: 060)", "course": "HIST 1302", "LD": 3, "UD": "N/A"}, {"title": "Exploration of the Humanities (Core: 040)", "course": "HUMA 1301", "LD": 3, "UD": "N/A"}, {"title": "Exploration of the Arts (Core: 050)", "course": "MUSI 1306", "LD": 3, "UD": "N/A"}, {"title": "Social Behavioral Science (Core: 080)", "course": "ECON 2301", "LD": 3, "UD": "N/A"}], " ELECTIVES (10 Hours - Requires Approval from Advisor)": [{"title": "Free Elective", "course": "ECON 2302", "LD": 3, "UD": "N/A"}, {"title": "Free Elective", "course": "MATH 2415", "LD": 3, "UD": "N/A"}, {"title": "Free Elective ", "course": "MATH 2420", "LD": 3, "UD": "N/A"}, {"title": "Free Elective", "course": "MUSI 3120", "LD": 1, "UD": "N/A"}, {"title": "Freshmen Seminar", "course": "UNIV 1010", "LD": 0, "UD": "N/A"}, {"title": "Core Curriculum Assessment", "course": "UNIV 2020", "LD": 0, "UD": "N/A"}]}
```

Comment:
Eventually, we should make sure not to commit data like this directly into the repo since it can be pretty large, but this works for now.